src/module.c

```diff
@@ -1,6 +1,6 @@
 #include <node_api.h>
 #include <napi-macros.h>
-#include <libmtp.h>
+#include "libmtp.h"
```

Comment:
Does `sudo apt-get install libmtp-dev` not install `libmtp.h`? If so, why do you need to include it in this commit?

Reply:
Because on macOS I'm including the library in `lib/`, so that the client machine doesn't have to do a `brew install libmtp` first. When we eventually have a Linux version of the Uploader, I should probably include the `libmtp` library for Linux as well.